src/server.rs

```diff
@@ -861,6 +989,17 @@ impl Server {
                     }
                 }
 
+                'S' => {
```

Comment:
Aren't these only sent on server startup? This method is only used for receiving query data I believe.

Reply:
These are also sent whenever we have set statements during a session that affect the parameters
eg.
```
F {"Type":"Query","String":"SET TimeZone TO 'Etc/UTC';"}
B {"Type":"CommandComplete","CommandTag":"SET"}
B {"Type":"ParameterStatus","Name":"TimeZone","Value":"Etc/UTC"}
B {"Type":"ReadyForQuery","TxStatus":"I"}

F {"Type":"Query","String":"SET TimeZone TO 'Etc/UTC';"}
B {"Type":"CommandComplete","CommandTag":"SET"}
B {"Type":"ReadyForQuery","TxStatus":"I"}
```
Also note that the second time a parameter status wasn't sent cause it was a no-op.

However for something like set statement timeout it's not sent
```
F {"Type":"Query","String":"set statement_timeout to 1000;"}
B {"Type":"CommandComplete","CommandTag":"SET"}
B {"Type":"ReadyForQuery","TxStatus":"I"}
```